Frontend/src/components/Table/Row.svelte

```diff
@@ -13,6 +14,38 @@
   };
 </script>
 
+<tr
+  on:click
+  style={`--rowHeight: ${rowHeight}px; ${
+    rowBackgroundColorFunction
+      ? "background-color: " + rowBackgroundColorFunction(item)
+      : ""
+  }`}>
+  {#each columns as col}
+    <td>
+      {#if col.isImageUrl}
+        {#if item[col.key] && displayValue(col, item) !== ""}
+          <img src={displayValue(col, item)} alt="item" />
+        {/if}
+      {:else if cellStyleFunction}
```

Comment:
```suggestion
      {:else}
```
cellStyleFunction will always be defined because it has a default value now